### ISSUE TITLE
Text contrast too low in ProductCardDescription

### ISSUE THREAD

**Issue by rpaulucci3** [nukeop/react-ui-cards#26]
Currently, the text in `ProductCardDescription` has a color of `#95A5A6`. This does not meet WCAG  requirements ([source](https://contrastchecker.com/?c=95a5a6&b=ecf0f1)).

I propose changing the text color in `ProductCardDescription` to `#2C3E50`, to match the product title and for WCAG compliance ([source](https://contrastchecker.com/?c=2c3e50&b=ecf0f1
)).
